src/com/amazon/ion/benchmark/schema/constraints/CodepointLength.java

```diff
@@ -0,0 +1,23 @@
+package com.amazon.ion.benchmark.schema.constraints;
+
+import com.amazon.ion.IonValue;
+
+public class CodepointLength extends QuantifiableConstraints {
```

Comment:
> Are we getting any value from having concrete subclasses (e.g. `CodepointLength`, `ByteLength`) that don't add any fields or methods to `QuantifiableConstraints`? Or would it be cleaner to have a single subclass for all of these, which includes an identifying keyword/enum? Or, is it anticipated that fields and/or methods will need to be added to these in the future?

For the subclasses which extend `QuantifiableConstraints`, some of them have different domain knowledge (e.g. `TimestampPrecision` has different definition of `min` and `max` from the other constraint subclasses, and this requires an additional method to process). And for the rest of subclasses which don't have additional fields or methods, if we keep them it will be easier to update if there are potential updates in these constraints.

Reply:
How many subclasses of QuantifiableConstraints actually add anything right now? Could additional subclasses be added if/when needed? Is there anything that makes doing that difficult?

Reply:
> How many subclasses of QuantifiableConstraints actually add anything right now? Could additional subclasses be added if/when needed? Is there anything that makes doing that difficult?


Based on the offline discussion, the changes were added to the commit https://github.com/amzn/ion-java-benchmark-cli/pull/34/commits/870c7a90f470f25d744871329f2f67e96dffc59e.